compass/landice/tests/enthalpy_benchmark/A/namelist.landice2

```diff
@@ -1,5 +1,5 @@
 config_surface_air_temperature_value = 268.15
 config_do_restart = .true.
-config_start_time = '100000-01-01_00:00:00'
+config_start_time = '100001-01-01_00:00:00'
```

Comment:
Is this change necessary?

Reply:
Yes - this is a restart after 10,000 years of the first phase.  Now that the first phase start time has moved from year 0 to 1, this restart start time needs to be advanced one year as well.